content/tutorials/history.textile

```diff
@@ -46,6 +46,9 @@ By default, channels will only store messages in memory for up to two minutes. H
 
 Using our "history API":/realtime/history is trivial.  Let's get started.
 
+bq. You can now persist the last realtime message on a channel for 365 days. *State Persistence* allows you to instantly retrieve realtime messages sent in the past, helping with  long-term state synchronization. "Read how to enable State Persistence":https://www.ably.io/documentation/realtime/channels/channel-parameters/rewind#persist-last-message.
```

Comment:
As before, extra space, and link format can be `/realtime/channels/channel-parameters/rewind#persist-last-message` rather than using full domain name.